packages/salesforcedx-vscode-core/src/services/sdr/componentSetUtils.ts

```diff
@@ -24,9 +24,11 @@ export async function setApiVersion(componentSet: ComponentSet): Promise<void> {
     return;
   }
 
-  // If no user-configured Api Version is present, then get the version from the Org.
-  const orgApiVersion = await getOrgApiVersion();
-  componentSet.apiVersion = orgApiVersion ?? componentSet.apiVersion;
+  // If no user-configured API Version is present, then get the version from the org.
+  const orgApiVersion = await exports.getOrgApiVersion();
+  componentSet.apiVersion = orgApiVersion && orgApiVersion.length > 0
```

Comment:
It turns out that `orgApiVersion ?? componentSet.apiVersion` doesn't work for an empty string.  (found this out while writing the unit tests, so unit tests "For The Win", eh?